src/internal/packager2/layout/package.go

```diff
@@ -179,12 +179,30 @@ func (p *PackageLayout) GetImageDir() string {
 }
 
 func (p *PackageLayout) Archive(ctx context.Context, dirPath string, maxPackageSize int) error {
```

Comment:
Tests are failing because `dirPath` will be empty when the `--output` flag is not used so we're running `os.MkdirAll` on an empty string

Reply:
Ah! Definitely a gap in input validation. Moving the parent directory creation down a level of abstraction helps with that problem